app/crude.py

```diff
@@ -1,30 +1,33 @@
+from typing import List
+from typing import Optional
+
 from model import User
 from sqlalchemy.orm import Session
 
 
-def get_user(db: Session, user_id: int):
+def get_user(db: Session, user_id: int) -> Optional[User]:
     return db.query(User).filter(User.id == user_id).first()
 
 
-def get_users(db: Session, skip: int = 0, limit: int = 100):
+def get_users(db: Session, skip: int = 0, limit: int = 100) -> List[User]:
```

Comment:
```suggestion
def get_users(db: Session, skip: int = 0, limit: int = 100) -> list[User]:
```

3.9 버전 이상에서는 저렇게 쓸 수 있습니다!